tools/maf_cleanup.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python
+import optparse
+
+### start of parsing parameters
+parser = optparse.OptionParser()
+parser.add_option('-i', '--input-file', action = 'store', dest = 'inputfile', help = "file name of input maf")
+parser.add_option('-o', '--output-file', action= 'store', dest = "outputfile", default=False, help = "name of output maf")
+
+(options, args) = parser.parse_args()
+input_file = options.inputfile
+output_file = options.outputfile
+
+if not input_file:
+	print 'ERROR: missing input MAF name.'
+	parser.print_help()
+elif not output_file:
+	print 'ERROR: missing output MAF name.'
+	parser.print_help()
+
+### end of parsing parameters
+
+### start of searching for header row
+input_maf = open (input_file, "r")
+header_line = ""
+for line in input_maf:
+	values = line.split("\t")
+	if values[0].upper() == "HUGO_SYMBOL":
+		header_line = line
+
+headers = header_line.rstrip().split("\t")
```

Comment:
rstrip needs a "\n" argument .. to prevent stripping tabs at end of line

Reply:
headerline wouldn't matter